scripts/release.sh

```diff
@@ -43,9 +42,14 @@ function tag_images_in_yaml() {
 # If $TAG is not empty, also copy it to $RELEASE_GCS_BUCKET bucket's "previous" directory.
```

Comment:
Update documentation

Reply:
The documentation is up-to-date. What am I missing?

Reply:
Hmmm... Maybe you mean the documentation about `tag_images_in_yaml()`?

Reply:
Updated the documentation about tag_images_in_yaml().

Reply:
Sorry, I meant publish_yaml. It only has $1 param in doc and you are also using $2

Reply:
Read the code again, you missed the internal helper function `verbose_gsutil_cp()`.

Reply:
oops. did not realize that it was an internal function